menas/ui/service/EntityValidationService.js

```diff
@@ -71,4 +71,26 @@ var EntityValidationService = new function () {
     return isOk;
   };
 
+  this.hasValidProperties = function(aProperties) {
+    let isOk = true;
+
+    aProperties.map((oProp) => {
+      //check essentiality - if Mandatory property has no or empty value, we fail
+      if(oProp.essentiality._t === "Mandatory" && !oProp.value) {
+        isOk = false;
+        oProp.validation = "Error";
+        oProp.validationText = `Provide valid ${oProp.name} - ${oProp.description}`;
```

Comment:
This might look a bit weird if no description was given (`propName1 - `), so maybe the string could be different in that case.

Reply:
Yeah changed that slightly